source/install.txt

```diff
@@ -6,10 +6,81 @@ Install the MongoDB for IntelliJ Plugin
 
 .. default-domain:: mongodb
 
+.. facet::
+   :name: genre
+   :values: tutorial
+
+.. meta:: 
+   :description: Install the MongoDB for IntelliJ Plugin from the JetBrains Marketplace.
+
 .. contents:: On this page
    :local:
    :backlinks: none
    :depth: 1
    :class: singlecol
 
-Content on this page will be addressed in DOCSP-44808.
+To install the {+intellij-full+} from the JetBrains Marketplace, follow the 
+steps on this page. 
+
+Prerequisites
+-------------
+
+To install the {+intellij-short+}, ensure that you install IntelliJ IDEA 
+Ultimate 2024.1.1 or later.
+
+To check your current version of IntelliJ IDEA, open the 
+:guilabel:`IntelliJ IDEA` system menu and click :guilabel:`About IntelliJ IDEA`.
+
+Steps 
+-----
+
+.. procedure:: 
+   :style: normal 
+
+   .. step:: Open the Settings menu 
+
+      Open the :guilabel:`IntelliJ IDEA` system menu and click 
+      :guilabel:`Settings`.
+
+      Alternatively, you can press :kbd:`Cmd` + :kbd:`,` on macOS or 
+      :kbd:`Ctrl` + :kbd:`Alt` + :kbd:`S` on Windows and Linux.
+
+   .. step:: Click on the Plugins tab 
+
+      In the left-hand navigation column, click :guilabel:`Plugins`.
+
+   .. step:: Open the Custom Plugin Repositories modal 
+
+      Next to the :guilabel:`Installed` tab, click the gear icon to open a 
```

Comment:
[suggestion]

Consider reordering to start with: "Click the gear icon next to...".

Reply:
Gotcha, that sounds much clearer/smoother! Updated.